src/Microsoft.ML.StaticPipe/NormalizerStaticExtensions.cs

```diff
@@ -16,61 +16,61 @@ namespace Microsoft.ML.StaticPipe
     /// </summary>
     public static class NormalizerStaticExtensions
     {
-        private const long MaxTrain = NormalizingEstimator.Defaults.MaxTrainingExamples;
+        private const long MaxTrain = NormalizingEstimator.Defaults.MaximumExampleCount;
         private const bool FZ = NormalizingEstimator.Defaults.FixZero;
```

Comment:
Can we in-line these and just break lines in the code?

Reply:
Sure.

---
In reply to: [264868132](https://github.com/dotnet/machinelearning/pull/2888#discussion_r264868132) [](ancestors = 264868132)